Gems/Atom/Feature/Common/Code/Source/PostProcessing/EditorModeFeedbackPassBase.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Console/IConsole.h>
```

Comment:
Why not use the registry system? the system already exists and can be utilized here instead of implementing a new temporary macro system

Reply:
Great suggestion, that's ultimately where these settings should live. For now, this is a temporary workaround for the lack of UI for configuration to allow for fast iteration of effect tuning. We will remove these CVARS and use the registry system as per your suggestion in the next phase of development 👍